test/unit/contextRequests/blackButtonCreationRequest.json

```diff
@@ -34,9 +34,24 @@
           "name": "op_extra",
           "value": "1234",
           "type": "string"
+        },
+        {
+          "name": "aux_op_status",
+          "value": "PENDING",
```

Comment:
In fact, it is already documented as an issue at https://github.com/telefonicaid/iotagent-thinking-things/issues/50 :)